examples/lotssdr2/Create_LoTTSDataset.ipynb

```diff
@@ -1,64 +1,208 @@
-    "    transform=transforms,\n",
-    ")\n",
-    "\n",
-    "for i in range(len(data)):\n",
-    "    if i > 10:\n",
-    "        break\n",
-    "    data.plot(\n",
-    "        i,\n",
-    "        contours=True,\n",
-    "        sigma_name=\"Isl_rms\",\n",
-    "        min_sigma=2,\n",
-    "        title=data.df.iloc[i][\"Source_Name\"] + data.df.iloc[i][\"S_Code\"],\n",
-    "    )\n",
-    "\n",
-    "data.df.head()"
-   ]
-  }
- ],
- "metadata": {
-  "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
-   "language": "python",
-   "name": "python3"
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "view-in-github"
+      },
+      "source": [
+        "<a target=\"_blank\" href=\"https://colab.research.google.com/github/mb010/Cata2Data/blob/main/examples/lotssdr2/Create_LoTTSDataset.ipynb\">\n",
+        "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+        "</a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0p1nWoeAcBL1"
+      },
+      "source": [
+        "## Create a LoTTS Dataset Using Cata2Data"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "IewboBsacOya"
+      },
+      "source": [
+        "To start, create a local clone of this repository\n",
+        "\n",
+        "Install cata2data into your local environment (We recommend that you should use a venv on your local machine).\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4OjNn-aHGrye"
+      },
+      "outputs": [],
+      "source": [
+        "!git clone https://github.com/mb010/Cata2Data.git && pip install ./Cata2Data && cp Cata2Data/examples/lotssdr2/data.py ."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "hszOwcQicuL7"
+      },
+      "source": [
+        "## Download the data\n",
+        "\n",
+        "Use the `data_scrapper.py` script to download the image files. If you want to just download one pointing (instead of all 841 pointings; 434 GB), then call it using the --test flag:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "lAC3ao-_H36q"
+      },
+      "outputs": [],
+      "source": [
+        "%%python Cata2Data/examples/lotssdr2/data_scrapper.py --dir downloaded_data/ --test"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nRpXlcpgf3y4"
+      },
+      "source": [
+        "This will have downloaded a .fits image file"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "bEKXzeCNf3ZZ"
+      },
+      "outputs": [],
+      "source": [
+        "!ls downloaded_data/public/DR2/mosaics/P000+23/"
```

Comment:
Can we assume that the 'first line' (i.e. the file downloaded with the `--test` flag will always be the same exact pointing? Or should this be a `*` instead of `P000+23`?